app/assets/stylesheets/to_change_everything/common/_font_styling.scss

```diff
@@ -0,0 +1,223 @@
+$black-text-shadow: 0 0 2rem #444;
+$header-font-stack: 'Vitesse SSm A', 'Vitesse SSm B', sans-serif !default;
+$body-font-stack:   'Whitney SSm A', 'Whitney SSm B', serif !default;
+
+@mixin title-text($font-size, $line-height)    { font: normal normal 800 $font-size /#{$line-height} $header-font-stack; }
+@mixin subtitle-text($font-size, $line-height) { font: normal normal 300 $font-size /#{$line-height} $header-font-stack; }
+@mixin pdfvideo-text($font-size, $line-height) { font: normal normal 600 $font-size /#{$line-height} $header-font-stack; }
+@mixin intro-header($font-size, $line-height)  { font: normal normal 500 $font-size /#{$line-height} $header-font-stack; }
+@mixin intro-text($font-size, $line-height)    { font: normal normal 600 $font-size /#{$line-height} $body-font-stack;   }
+@mixin imagetext($font-size, $line-height)     { font: normal normal 300 $font-size /#{$line-height} $header-font-stack  }
+@mixin body-text($font-size, $line-height)     { font: normal normal 500 $font-size /#{$line-height} $body-font-stack    }
+@mixin outro-text($font-size, $line-height)    { font: normal normal 600 $font-size /#{$line-height} $body-font-stack    }
+@mixin takeflight($font-size, $line-height)    { font: normal normal 800 $font-size /#{$line-height} $header-font-stack  }
+@mixin anarchy-text($font-size, $line-height)  { font: normal normal 600 $font-size /#{$line-height} $body-font-stack    }
+@mixin next-text($font-size, $line-height)     { font: normal normal 300 $font-size /#{$line-height} $header-font-stack  }
```

Comment:
Funny timing!

Just today at work, we ran into a weird ass issue with Heroku + Asset Pipeline + `font:` one-liners with `normal` in it.

The short and long of it is, don't one line `font` + `normal`. Weird things happen. It seems like Heroku's Asset Pipeline config is converting `normal` into `400`, but that causes unexpected behavior.

The work around was to do all of the `font-*` properties as their own rule.